src/main/resources/view/LightTheme.css

```diff
@@ -367,3 +367,27 @@
     -fx-font-size: 13px;
     -fx-text-fill: black;
 }
+
+#deadline {
+    -fx-font-family: "Segoe UI Semibold";
+    -fx-font-size: 14px;
+    -fx-text-fill: tomato;
```

Comment:
maybe only set the text to red when the time has passed like you have already done in appointment and property card?